playbooks/roles/cluster/templates/workflow_survey.json

```diff
@@ -1,51 +1,84 @@
-        "type": "text"
-      },
-      {
-        "question_description": "AWS account name",
-        "min": 0,
-        "default": "fheng.AWS",
-        "max": 100,
-        "required": false,
-        "choices": "{{ cluster_install_survey_aws_accounts }}",
-        "variable": "cluster_credential_bundle_aws_name",
-        "question_name": "AWS account name",
-        "type": "text"
-      },
-      {
-        "question_description": "Region you wish to provision the cluster in",
-        "min": null,
-        "default": "",
-        "max": null,
-        "required": true,
-        "choices": "us-east-1\neu-west-1\neu-west-2",
-        "variable": "oo_sublocation",
-        "question_name": "AWS Region",
-        "type": "multiplechoice"
-      },
-      {
-        "question_description": "Flag for provisioning logging stack",
-        "min": null,
-        "default": "",
-        "max": null,
-        "required": true,
-        "choices": "true\nfalse",
-        "new_question": true,
-        "variable": "survey_provision_logging",
-        "question_name": "Provision Logging",
-        "type": "multiplechoice"
-      }
-    ]
-  }
\ No newline at end of file
+  "description": "",
+  "name": "",
+  "spec": [
+    {
+      "question_description": "Desired name of Openshift cluster",
+      "min": 0,
+      "default": "",
+      "max": 16,
+      "required": true,
+      "choices": "",
+      "variable": "oo_clusterid",
+      "question_name": "Cluster Name",
+      "type": "text"
+    },
+    {
+      "question_description": "AWS account name",
+      "min": 0,
+      "default": "fheng.AWS",
+      "max": 100,
+      "required": false,
+      "choices": "{{ cluster_install_survey_aws_accounts }}",
+      "variable": "cluster_credential_bundle_aws_name",
+      "question_name": "AWS account name",
+      "type": "text"
+    },
+    {
+      "question_description": "Region you wish to provision the cluster in",
+      "min": null,
+      "default": "",
+      "max": null,
+      "required": true,
+      "choices": "us-east-1\neu-west-1\neu-west-2",
+      "variable": "oo_sublocation",
+      "question_name": "AWS Region",
+      "type": "multiplechoice"
+    },
+    {
+      "question_description": "Flag for provisioning logging stack",
+      "min": null,
+      "default": "",
+      "max": null,
+      "required": true,
+      "choices": "true\nfalse",
+      "new_question": true,
+      "variable": "survey_provision_logging",
+      "question_name": "Provision Logging",
+      "type": "multiplechoice"
+    },
+    {
+      "question_description": "Number of master nodes required",
+      "min": 1,
+      "default": 3,
+      "max": 9,
+      "required": true,
+      "new_question": true,
+      "variable": "openshift_aws_master_group_desired_size",
+      "question_name": "Master Nodes",
+      "type": "integer"
+    },
+    {
+      "question_description": "Number of infra nodes required",
+      "min": 2,
```

Comment:
I think with the current provisioning tooling we need a minimum of 3 infra nodes